test/distributed/_tensor/test_colletive_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+import torch
+from torch.distributed._tensor import DeviceMesh
+from torch.distributed._tensor._collective_utils import shard_dim_alltoall
+
+from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.distributed._tensor.common_dtensor import (
+    DTensorTestBase,
+    with_comms,
+)
+
+
+class TestCollectiveUtils(DTensorTestBase):
+    @property
+    def world_size(self):
+        return 4
+
+    @with_comms
+    def test_shard_dim_alltoall(self):
```

Comment:
can you add this test case to `test_redistribute.py` instead of creating a separate test file?